deps/rabbit/src/rabbit_reader.erl

```diff
@@ -645,7 +645,22 @@ handle_other(handshake_timeout, State) ->
 handle_other(heartbeat_timeout, State = #v1{connection_state = closed}) ->
     State;
 handle_other(heartbeat_timeout,
-             State = #v1{connection = #connection{timeout_sec = T}}) ->
+             State = #v1{connection = #connection{timeout_sec = T, name = ConnectionName},
+                         sock = Socket}) ->
+    case ssl:connection_information(Socket, [keylog]) of
```

Comment:
is this related to message interceptors?

Reply:
Definitely not, my bad.